Gemfile

```diff
@@ -11,6 +11,7 @@ gem "dynamoid"
 gem "govuk_app_config", "~> 2.1.1"
 gem "govuk_publishing_components", "~> 21.31.0"
 gem "log_formatter", "~> 0.8.2"
```

Comment:
Might be able to remove log_formatter and use lograge's JSON thing? I tried using lograge originally, but I couldn't get it to work - I ended up raging and using this other thing.

Reply:
This worked for me locally after removing `log_formatter` from the Gemfile:

```
config.lograge.enabled = true
config.lograge.formatter = Lograge::Formatters::Json.new
```

This gives the following log line:

```
{"method":"GET","path":"/live-in-england","format":"html","controller":"CoronavirusForm::LiveInEnglandController","action":"show","status":200,"duration":484.42,"view":483.82}
```

I thought you might need to ignore `/healthcheck` but looks like lograge already ignores that because it doesn't find a controller.

Reply:
🤦‍♂ I think I might have forgotten `config.lograge.enabled = true` last time I tried this. I was tired.

Reply:
Ah I guess I'm a little confused what output we need - do we need JSON? it didn't appear to  logging in JSON before 🤔 (or maybe it was)

Reply:
But I'll go ahead and remove this formatter anyway

Reply:
> Ah I guess I'm a little confused what output we need - do we need JSON? it didn't appear to logging in JSON before 🤔 (or maybe it was)

I think currently it's not logging anything but errors due to the setting here:
https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/blob/2eef96ff556d685b649919baceff20c50233b0a5/config/initializers/dynamoid.rb#L11

That's mutating the level on the default `Rails.logger`.

If the logs are going into Splunk then the default key/value pairs is the best option:
https://dev.splunk.com/enterprise/docs/developapps/logging/loggingbestpractices/#Use-clear-key-value-pairs

Reply:
The logs are being shipped to Splunk and [Elasticsearch / Logstash / Kibana], so JSON would be nice (otherwise we'll have to change our Logstash filters I think). It was using `log_formatter/ruby_json_formatter` before, but that was only because I couldn't get lograge to do the right thing (and that was probably just me being stupid).

We might also want to consider not shipping to ELK though - not sure how useful it is given we built most of our dashboards in splunk.

Reply:
There's an explicit Logstash formatter for lograge if that's what's needed:
https://github.com/roidrage/lograge/blob/master/lib/lograge/formatters/logstash.rb

(note the requirement for the logstash-event gem)